RecoEcal/Configuration/python/RecoEcal_EventContent_cff.py

```diff
@@ -107,11 +41,54 @@
-        e.toModify( ec, func=lambda outputCommands: outputCommands.extend(['keep recoSuperClusters_correctedIslandBarrelSuperClusters_*_*',
-                                                                           'keep recoSuperClusters_correctedIslandEndcapSuperClusters_*_*'
-                                                                           ])
-                    )
-    for ec in [RecoEcalRECO.outputCommands, RecoEcalFEVT.outputCommands]:
-        e.toModify( ec, func=lambda outputCommands: outputCommands.extend(['keep recoCaloClusters_islandBasicClusters_*_*'])
-                    )
+    e.toModify( RecoEcalAOD.outputCommands, 
+                func=lambda outputCommands: outputCommands.extend(['keep recoSuperClusters_correctedIslandBarrelSuperClusters_*_*',
+                                                                   'keep recoSuperClusters_correctedIslandEndcapSuperClusters_*_*'])
+              )
+# RECO content
+RecoEcalRECO = cms.PSet(
+    outputCommands = cms.untracked.vstring(
+	# Barrel clusters
+        'keep *_hybridSuperClusters_*_*',
+        'keep recoSuperClusters_correctedHybridSuperClusters_*_*',
+	# Endcap clusters
+        'keep *_multi5x5SuperClusters_*_*',
+        'keep recoSuperClusters_multi5x5SuperClusters_*_*',
+        'keep recoSuperClusters_multi5x5SuperClustersWithPreshower_*_*',
+        # Particle Flow superclusters
+        'keep *_particleFlowSuperClusterECAL_*_*',
+        'keep *_particleFlowSuperClusterOOTECAL_*_*',
+	# DROP statements # not used
+        #'drop recoClusterShapes_*_*_*', 
+        #'drop recoBasicClustersToOnerecoClusterShapesAssociation_*_*_*',
+        #'drop recoBasicClusters_multi5x5BasicClusters_multi5x5BarrelBasicClusters_*',
+        #'drop recoSuperClusters_multi5x5SuperClusters_multi5x5BarrelSuperClusters_*')
+        )
+)
+RecoEcalRECO.outputCommands.extend(RecoEcalAOD.outputCommands)
+_phase2_hgcal_scCommands = ['keep *_particleFlowSuperClusterHGCal_*_*',
+                            'keep *_particleFlowSuperClusterHGCalFromMultiCl_*_*']
+phase2_hgcal.toModify(RecoEcalRECO,
+    outputCommands = RecoEcalRECO.outputCommands + _phase2_hgcal_scCommands)
+
+for e in [pA_2016, peripheralPbPb, pp_on_AA_2018, pp_on_XeXe_2017, ppRef_2017]:
+    e.toModify( RecoEcalRECO.outputCommands,
+                func=lambda outputCommands: outputCommands.extend(['keep recoCaloClusters_islandBasicClusters_*_*'])
+              )
+# Full Event content 
+RecoEcalFEVT = cms.PSet(
+    outputCommands = cms.untracked.vstring(
+	# Hits
+	'keep *_reducedEcalRecHitsEB_*_*',
+        'keep *_reducedEcalRecHitsEE_*_*',
+        'keep *_reducedEcalRecHitsES_*_*', 
+        'keep *_interestingEcalDetId*_*_*', 
+        'keep *_ecalWeightUncalibRecHit_*_*', 
+        'keep *_ecalPreshowerRecHit_*_*', 
+	# Barrel clusters
+        'keep *_correctedHybridSuperClusters_*_*',
+	# Endcap clusters
+        'keep *_multi5x5*_*_*',
+        'keep *_correctedMulti5x5*_*_*')
```

Comment:
Is there anything else beside ```correctedMulti5x5SuperClustersWithPreshower``` already saved in the AOD event content?

Reply:
No, only `vector<reco::SuperCluster>            "correctedMulti5x5SuperClustersWithPreshower"   ""                "RECO"` exists in AOD event content (also in RECO/FEVT event contents). So this line is redundant so it can be deleted.